src/xSTIR/cSTIR/include/sirf/STIR/stir_data_containers.h

```diff
@@ -1381,6 +1401,13 @@ namespace sirf {
 		void semibinary_op(const DataContainer& a_x, float y, float(*f)(float, float));
 		void binary_op(const DataContainer& a_x, const DataContainer& a_y, float(*f)(float, float));
 
+		size_t address() const {
+		    return reinterpret_cast<size_t>(_data->get_const_full_data_ptr());
+		}
+		size_t address() {
+		    return reinterpret_cast<size_t>(_data->get_full_data_ptr());
+		}
```

Comment:
can always use the `const` version here as well

Reply:
done